### PR TITLE
Feat vue docs

### PR DESCRIPTION
**Pull request by AlexTraher**
Update Nuxt to the stable v3 version and updates some of the docs for the nuxt and vue packages.

### CODE REVIEW

File: packages/vue/README.md [hunk 1]
```diff
@@ -100,3 +104,126 @@ type useSharedState = <
...
+
+Read [@leanjs/core](../core/README.md#basic-usage) if you have not already created your own `createRuntime` function
+
+:::
+
+## Components
+
+### `HostProvider`
+
+You have to **call [createRuntimeBindings](#createruntimebindings) to create a `HostProvider` component** before you use it. `HostProvider` stores in a React context values that are shared across apps hosted in the same component tree. Props:
```
Comment (alexlbr) [inline packages/vue/README.md:195]: `HostProvider stores in a **Vue** context

File: packages/vue/README.md [hunk 2]
```diff
@@ -100,3 +104,126 @@ type useSharedState = <
...
+    context: { appName: "AppExample" },
+  });
+
+  const origin = process.env.MICROFRONTENDS_ORIGIN;
+</script>
+```
+
+### `Host`
+
+It hosts a composable app in a React host.
```
Comment (alexlbr) [inline packages/vue/README.md:227]: It hosts a composable app in a **Vue** host.

**Review by alexlbr — CHANGES_REQUESTED** [review]
Thank you Alex!

File: packages/vue/README.md (hunk 2 above)
Comment (AlexTraher) [reply packages/vue/README.md]: done

File: packages/vue/README.md (hunk 1 above)
Comment (AlexTraher) [reply packages/vue/README.md]: done 😄

**Review by alexlbr — APPROVED** [review]

**Pull request merged**